SU-SWS/decanter
# Switch Source Serif Pro to import from Google Fonts

Switch Source Serif Pro to import from Google Fonts instead of hosted files.
- Update `_fonts.scss` with import copy/pasted from Google Fonts, remove hosted file imports.
- Run `publish` to generate compiled decanter.css file.

# READY FOR REVIEW

# Summary
Import all versions of Source Serif Pro from Google Fonts (instead of hosted AWS files) now that they are available from Google. This maintains consistency with how the other fonts are imported to the project.

# Needed By (Date)
No specific deadline.

# Urgency
Not critical

# Steps to Test
1. Create tests file in the `developer` directory – you'll need `test.twig` and `test.scss` for this
2. In `test.twig` add markup for at least one heading and some body text. 
3. Make a portion of each the heading and the body italic by adding an `<em>` or `<i>` to a few words.
4. In `test.scss` add a rule so your content displays in the Source Serif font – `font-family: $su-font-serif;`
5. `npm run dev` to compile the assets.
6. `npm run start` to start the development server and open your test page.
7. Check your test page in the browser to ensure that it displays the proper serif font – Source Serif Pro – and ensure that the italics display appropriately.
8. Using dev tools, view the `decanter.css` file in the Sources tab. Line 2 should now be a longer string that includes all variations of Source Serif Pro.

# Affected Projects or Products
Just a helpful update, that's all.

# Associated Issues and/or People
I've discussed this change in the #decanter Slack channel with @yvonnetangsu and @sherakama. We agreed that it'd be worthwhile. This is my first PR to Decanter, so lemme know if I missed something. :)

# See Also
- [PR Checklist](https://gist.github.com/sherakama/0ba17601381e3adbe0cad566ad4d80a5)
